java-generic-mqtt/src/main/java/com/cumulocity/generic/mqtt/client/websocket/GenericMqttWebSocketSubscriber.java

```diff
@@ -0,0 +1,75 @@
+package com.cumulocity.generic.mqtt.client.websocket;
+
```

Comment:
issue: same issues and and other comments about Token scope and duration on Publisher also apply to Subscriber

Reply:
Its changed now.